cmd/data-usage-cache.go

```diff
@@ -62,9 +63,10 @@ type dataUsageEntryInfo struct {
 
 type dataUsageCacheInfo struct {
 	// Name of the bucket. Also root element.
-	Name       string
-	LastUpdate time.Time
-	NextCycle  uint8
+	Name        string
+	LastUpdate  time.Time
+	NextCycle   uint32
```

Comment:
Is this going to be a breaking change for existing data?

Reply:
No. Numbers are simply integers, so making the type longer is backwards compatible.